src/main/java/org/venice/piazza/servicecontroller/messaging/ServiceMessageThreadManager.java

```diff
@@ -15,229 +15,97 @@
-	@Value("${SPACE}")
-	private String SPACE;
-
-	@Autowired
-	private CoreServiceProperties coreServiceProperties;
 	@Autowired
 	private PiazzaLogger coreLogger;
 	@Autowired
-	ServiceMessageWorker serviceMessageWorker;
+	private ServiceMessageWorker serviceMessageWorker;
 	@Autowired
 	private ServiceTaskManager serviceTaskManager;
 	@Autowired
 	private AsyncServiceInstanceScheduler asyncServiceInstanceManager;
 
-	private static final Logger LOG = LoggerFactory.getLogger(ServiceMessageThreadManager.class);
-	private static final String TOPIC_FORMAT = "%s-%s";
-	
-	/**
-	 * Constructor for ServiceMessageThreadManager
-	 */
-	public ServiceMessageThreadManager() {
-		closed = makeAtomicBoolean();
-
-	}
-
-	/**
-	 * Initializing stuff
-	 */
-	@PostConstruct
-	public void initialize() {
-
-		// Initialize dynamic topic names
-		EXECUTE_SERVICE_JOB_TOPIC_NAME = String.format(TOPIC_FORMAT, (new ExecuteServiceJob()).getClass().getSimpleName(), SPACE);
-
-		topics = Arrays.asList(EXECUTE_SERVICE_JOB_TOPIC_NAME);
-
-		coreLogger.log("============================================================", Severity.INFORMATIONAL);
-		coreLogger.log("EXECUTE_SERVICE_JOB_TOPIC_NAME=" + EXECUTE_SERVICE_JOB_TOPIC_NAME, Severity.INFORMATIONAL);
-		coreLogger.log("KAFKA_GROUP=" + KAFKA_GROUP, Severity.INFORMATIONAL);
-		coreLogger.log("KAFKA_HOSTS=" + KAFKA_HOSTS, Severity.INFORMATIONAL);
-		coreLogger.log("============================================================", Severity.INFORMATIONAL);
-
-		/* Initialize producer and consumer for the Kafka Queue */
-		producer = KafkaClientFactory.getProducer(KAFKA_HOSTS);
-		consumer = KafkaClientFactory.getConsumer(KAFKA_HOSTS, KAFKA_GROUP);
-
-		// Initialize the HashMap
-		runningServiceRequests = new HashMap<String, Future<?>>();
-
-		// Start polling for Kafka Jobs on the Group Consumer.. This occurs on a
-		// separate Thread so as not to block Spring.
-		Thread kafkaListenerThread = new Thread() {
-			@Override
-			public void run() {
-				pollServiceJobs();
-			}
-		};
-
-		// Subscribe for the topics
-		consumer.subscribe(topics);
-		kafkaListenerThread.start();
+	@Value("${SPACE}")
+	private String SPACE;
 
-		// Start polling for Kafka Abort Jobs on the unique Consumer.
-		Thread pollAbortThread = new Thread() {
-			@Override
-			public void run() {
-				pollAbortServiceJobs();
-			}
-		};
-		pollAbortThread.start();
-	}
+	private ObjectMapper mapper = new ObjectMapper();
+	private static final Logger LOG = LoggerFactory.getLogger(ServiceMessageThreadManager.class);
+	private Map<String, Future<?>> runningServiceRequests = new HashMap<String, Future<?>>();
 
 	/**
-	 * Polls for service controller topics and handles these requests.
+	 * Processes a message for a request to execute a service job
+	 * 
+	 * @param serviceJobRequest
+	 *            The ExecuteServiceJob Request with the Execution information
 	 */
-	public void pollServiceJobs() {
-
+	@RabbitListener(queues = "ExecuteServiceJob-${SPACE}")
```

Comment:
I see queues "ExecuteServiceJob-SPACE" and "Abort-Job-SPACE". I would pick a naming convention and use it throughout (one has an additional hyphen).